kotlin/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.kt

```diff
@@ -62,7 +63,10 @@ class CheckoutActivity : AppCompatActivity() {
 
         // Setup buttons
         googlePayButton = layout.googlePayButton
-        googlePayButton.initialize(ButtonOptions.newBuilder().setAllowedPaymentMethods("VISA").build())
+        googlePayButton.initialize(
+            ButtonOptions.newBuilder()
+                .setAllowedPaymentMethods(Constants.SUPPORTED_NETWORKS.joinToString(",")).build()
```

Comment:
The `string` we pass to the `setAllowedPaymentMethods` needs to be a valid payload like here: 

https://github.com/google-pay/android-quickstart/blob/cf48a7ba4b797b7f76c91fb0629577e06654c627/kotlin/app/src/main/java/com/google/android/gms/samples/wallet/util/PaymentsUtil.kt#L106-L121